src/BaGet.Core.Server/BaGet.Core.Server.csproj

```diff
@@ -1,16 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Would it be better to use `<FrameworkReference Include="Microsoft.AspNetCore.App" />` instead? With the framework reference you won't need an entry point in `OperationCancelledMiddleware.cs`